src/scenes/NurseryRouter/NurseryWithdrawalsTable.tsx

```diff
@@ -86,6 +103,38 @@ export default function NurseryWithdrawalsTable(): JSX.Element {
     [filterColumns, filterOptions, filters]
   );
 
+  const featuredFilters: FeaturedFilterConfig[] = useMemo(
```

Comment:
thoughts on a reusable hook to populate this (similar to what's in planting progress).. only if it makes sense to do so

Reply:
Good call, the only difference between the two current implementations is the `field` within the `searchNodeCreator`. I'll refactor it in a follow up PR